### DISCUSSION TITLE
Testing my SSH key connection

### DISCUSSION THREAD

**Discussion by ghost** [community/community#29427] [Other Feature Feedback, Questions, & Ideas]
When I test my SSH key connection I get a "Bad configuration option: host*" 

followed by "terminating, 1 bad configuration options"

Any idea where I should start looking, I am a new programming student

**Comment by airtower-luna** [comment]
The message says that your SSH configuration is invalid, so you should check the configuration file. Usually it'll be in `~/.ssh/config`, with `~` representing your home directory.
